scripts/process-google-services.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.application'
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath 'com.google.gms:google-services:3.0.0'
+        classpath 'com.android.tools.build:gradle:1.5.0'
+    }
+
+}
+
+android{
+    compileSdkVersion 23
+    buildToolsVersion '23.0.3'
+
+    sourceSets {
+        main {
+            manifest.srcFile 'AndroidManifest.xml'
+        }
+    }
+}
+
```

Comment:
Is this somethat that could go to the other gradle file? Or do we need a new one for the services plugin ?